modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/classloading/ClassLoadingOnConnectionsTestCase.java

```diff
@@ -72,6 +82,12 @@ public void allOptionalParameterGroup() throws Exception {
     verifyUsedClassLoaders(ALL_OPTIONAL_PARAMETER_GROUP);
   }
 
+  @Test
+  public void operationWithMetadataResolver() throws Exception {
```

Comment:
I don't feel that this test belongs in this class.

Reply:
will refactor